saleor/graphql/checkout/tests/mutations/test_checkout_create.py

```diff
@@ -2791,3 +2791,46 @@ def test_checkout_create_triggers_webhooks(
     assert WebhookEventSyncType.CHECKOUT_CALCULATE_TAXES in sync_deliveries
     tax_delivery = sync_deliveries[WebhookEventSyncType.CHECKOUT_CALCULATE_TAXES]
     assert tax_delivery.webhook_id == tax_webhook.id
+
+
+def test_checkout_create_with_metadata(api_client, graphql_address_data, channel_USD):
```

Comment:
Maybe we should also add a test for providing only `private_metadata` and only `metadata`.

Also missing `given, when, then`.

Reply:
done